Documentation/ceph-cluster-crd.md

```diff
@@ -96,6 +98,16 @@ log enough to ignore network blips where mons are failed over too often.
 - `ROOK_MON_HEALTHCHECK_INTERVAL`: The frequency with which to check if mons are in quorum (default is 45 seconds)
 - `ROOK_MON_OUT_TIMEOUT`: The interval to wait before marking a mon as "out" and starting a new mon to replace it in the quroum (default is 600 seconds)
 
+**volumeClaimTemplate**. This field is optional, and when not provided, HostPath
```

Comment:
why not just include this with the bullet point above?

Reply:
i don't consider bullet points good locations for extended prose, but i also don't want to get too radical :) fixed